code/modules/client/preferences.dm

```diff
@@ -55,6 +62,8 @@ var/list/preferences_datums = list()
 	var/hair_color = "000"				//Hair color
 	var/facial_hair_style = "Shaved"	//Face hair type
 	var/facial_hair_color = "000"		//Facial hair color
+	var/beard_level_enabled = FALSE		//Enables beard level

```

Comment:
I'm gonna say no and then maybe make a future pr that forces you to grow your beard without any cool effects just for you goofball